Add info on troubleshooting Group properties not attribute Mixpanel groups (#6840)
segmentio/segment-docs: src/connections/destinations/catalog/actions-mixpanel/index.md

```js
analytics.track('Example Event', { custom_group_key : 'group1' });
```
### Group properties are not attributed to Mixpanel Groups

> If the Mixpanel (Actions) destination uses $group_id as the group key, ensure that the mappings handling your `Group` events have the field for **Group ID** mapped to a valid value and the field for **Group Key** is left blank or without value. By default, the field **Group ID** is map to the event variable `groupId` and leaving the field for **Group Key** as blank will use $group_id as the group key. 

Reviewer: ```suggestion
If the Mixpanel (Actions) destination uses `$group_id` as the group key, ensure that the mappings handling your `Group` events have the **Group ID** mapped to a valid value the **Group Key** is left blank or without value. By default, the **Group ID** maps to the event variable `groupId`. If you leave the **Group Key** blank, the `$group_id` is used as the group key. 
```
